api-test/bdd/steps.go

```
@@ -0,0 +1,596 @@
+	}
+	if s.adminToken != "" {
+		req.Header.Set("Authorization", "Bearer "+s.adminToken)
+	}
+
+	resp, err := client.Do(req)
+	if err != nil {
+		s.record(label, method, fullURL, req.Header, resolvedBody, 0, nil, nil)
+		return fmt.Errorf("%s %s: %w", method, fullURL, err)
+	}
+	respBody, _ := io.ReadAll(resp.Body)
+	resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)